src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpAuthHelper.cs

```diff
@@ -296,7 +296,7 @@ public void SaveServerCredentialsToCache(Uri uri, uint authScheme, ICredentials
                 Interop.WinHttp.WINHTTP_OPTION_AUTOLOGON_POLICY,
                 ref optionData))
             {
-                WinHttpException.ThrowExceptionUsingLastError();
+                WinHttpException.ThrowExceptionUsingLastError("WinHttpSetOption");
```

Comment:
nit: `nameof(Interop.WinHttp.WinHttpSetOption)`... same in other such situations.  It'll result in the same string, just safe against refactorings and the like.

Reply:
Will fix.